src/views/darkroom.c

```diff
@@ -1499,7 +1500,7 @@ static void _iso_12646_quickbutton_clicked(GtkWidget *w, gpointer user_data)
 
   if(d->iso_12646.enabled)
   {
-    d->border_size = 0.125 * d->width;
+    d->border_size = 0.125 * MIN(d->width, d->height);
```

Comment:
To be conservative with previous version it should be `MAX()` otherwise a picture gets a quite smaller border than in current master. Do you agree?

Reply:
I see, yet I would stay with MIN and use  something like 20%.

That fits better with wide monitors..

Reply:
Why not as long as the display is still useful as a color assessment mode. The size of the border really matters.

Reply:
I looked up the iso and some guidelines from museums for reproduction. They mention at least 1 inch of white border. What do you think?

Reply:
Yes 1 inch sounds right, that's also my recollection when discussing the integration.

Reply:
OK I will do that. Hard wired 1 inch or as a hidden conf?

Reply:
I will go hard wired for now. It will always be possible later to have it as an option even if I doubt it will be needed.